manual.md

```diff
@@ -659,23 +674,25 @@ Functions
 ---------
 
 Functions take parameters (normally
-between parentheses) and produce a
-result. Functions can be abbreviated by
-using a shorter name ended in a dot,
-for example you can write `R.(10)`
-instead of `RAND(10)`.
+between parentheses) and produce and
+return a result. Functions can be
```

Comment:
Probably we should remove the "and produce".

Reply:
Mened in https://github.com/dmsc/fastbasic/pull/59/commits/350fd46637313c45409f17825638d5d13b4996a3